geoips/geoips_utils.py

```diff
@@ -37,20 +36,6 @@ def remove_unsupported_kwargs(module, requested_kwargs):
     return requested_kwargs
 
 
-def get_entry_point_group(group):
-    """Get entry point group."""
-    # NOTE: When there is a .egg-info directory in the plugin package top
-    # level (ie, from setuptools pip install -e), it seems to return that
-    # package twice in this list.  For now, just use the full list with
-    # duplicates.
-    if sys.version_info[:3] >= (3, 10, 0):
-        eps = metadata.entry_points(group=group)
-    else:
-        eps = metadata.entry_points()[group]
-
-    return eps
-
```

Comment:
@jsolbrig I think this is ready for approval

Reply:
Yup! Sorry that I forgot to comment about that.